src/components/calcite-alert/calcite-alert.tsx

```diff
@@ -41,6 +43,9 @@ export class CalciteAlert {
   /** Is the alert currently active or not */
   @State() active: boolean = false;
 
+  /** Determine if the alert contains an alert-link slot */
+  @State() hasLink: boolean = false;
```

Comment:
unless you want to trigger a re-render when this changes, you'll want to omit the @State decorator.